.github/workflows/windows.yml

```diff
@@ -119,7 +119,7 @@ jobs:
 
       - name: Build and install
         run: |
-          python -m pip install . -v -Ccompile-args="-j2" -Csetup-args="-Dallow-noblas=true"
+          python -m pip install . -v -Ccompile-args="-j2" -Csetup-args="-Dallow-noblas=true" -Csetup-args="-Dc_args=/fp:contract" -Csetup-args="-Dcpp_args=/fp:contract"
```

Comment:
This would read better with a line continuation `\` to break the line. Probably for the previous use as well.

Reply:
This should be part of the meson flags when building SIMD code. I suggest moving it here: https://github.com/numpy/numpy/blob/b353a188204bcd6d483ac3ac87b87009485d36c3/meson_cpu/x86/meson.build#L214